### ISSUE TITLE
Make the start-urbit script more robust in the Docker image

### ISSUE THREAD

**Issue by lukebuehler** [urbit/urbit#4560]
I cannot use the `tloncorp/urbit` Docker image in Kubernetes because the the cloud mounted volume creates a `lost+found` directory in the `/urbit` folder. This results in the wrong directory being picked up in the final few lines of the [`start-urbit` script](https://github.com/urbit/urbit/blob/master/nix/pkgs/docker-image/default.nix). Specifically, the `$dirname` variable ends up being set to "lost+found," which of course results in the pier not booting at all.

It would be nice if we could come up with a more deterministic way of how this Docker image boots the pier.

**Comment by locpyl-tidnyd** [comment]
Also, when the `start-urbit` script sees a file with the `.comet` extension and boots a comet, it is supposed to create the new pier in a directory named as the comet file w/o extension, but it ends up creating it in a directory named `*` because of bash quoting problems. Tagging @eamsden (~ritpub-sipsyl) on his suggestion.

**Comment by eamsden** [comment]
@wexpertsystems having played with the images do you have any ideas on this?

**Comment by joemfb** [comment]
The pier name does not have to be the ship name, that's just the default. Use `-c foo` to make it be `foo`. It could even be hardcoded in the container.

**Comment by eamsden** [comment]
that is a good idea, actually. A hardcoded pier name would avoid the issues with the `lost+found` directory.

**Comment by wexpertsystems** [comment]
Yeah the way the script does this is not ideal. It also runs into issues if the key has a name like `sampel-palnet-1.key` which is how they download from bridge, you need to remove the numeric suffix for it to work.

The script just lists the directories and naively takes the first one. An easy fix for this "lost+found" issue might be to step through those directories and pick the first one with an `.urb` directory in it. And to probably disregard "lost+found" entirely.
